doc/lua_api.md

```diff
@@ -10248,7 +10248,7 @@ Used by `minetest.add_particle`.
 
     texture = "image.png",
     -- The texture of the particle
-    -- v5.6.0 and later: also supports the table format described in the
+    -- v5.9.0 and later: also supports the table format described in the
```

Comment:
technically the table format was fine, the properties just didn't do anything.
I would document that exactly as written

Reply:
I'd opt for:

```lua
-- The texture of the particle
-- v5.6.0 and later: also supports the table format described in the
-- following section, but due to a bug this did not take effect
-- (beyond the texture name).
-- v5.9.0 and later: fixes the bug.
```